channels.go

```diff
@@ -18,10 +18,7 @@ type channelResponseFull struct {
 
 // Channel contains information about the channel
 type Channel struct {
-	groupConversation
-	IsChannel bool `json:"is_channel"`
-	IsGeneral bool `json:"is_general"`
-	IsMember  bool `json:"is_member"`
+	GroupConversation
```

Comment:
I think you may have missed the point of adding GroupConversation. It includes the common members that are shared by both Channel and Group. You cannot be a member (`IsMember`) of a Channel. You cannot have a group that `IsGeneral`. `IsChannel` and `IsGroup` are mutually exclusive. Channels and Groups never come back in a unified list. Consequently, I'm not sure why you made the change?

Further, groupConversation shouldn't be exported.